src/browser-frontend/sagas/webcontents-action-handlers.js

```diff
@@ -94,8 +94,8 @@ function* onPageDidNavigateInternal() {
   // TODO
 }
 
-function* onPageDidNavigateToNewWindow() {
-  // TODO
+function* onPageDidNavigateToNewWindow({ payload: { pageId, url } }) {
+  yield put(PagesModelActions.addPage({ url, pageId }));
```

Comment:
I'm still passing it, but now as `parentId` to position the tab beside the parent